src/main/scala/com/databricks/spark/sql/perf/mllib/MLTransformerBenchmarkable.scala

```diff
@@ -0,0 +1,107 @@
+package com.databricks.spark.sql.perf.mllib
+
+import com.databricks.spark.sql.perf._
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.spark.sql._
+
+import scala.collection.mutable.ArrayBuffer
+
+class MLTransformerBenchmarkable(
```

Comment:
This is the mother of all issues with subclassing, and I think the jury is still out on this one. Since Estimator is a transformer with an extra fitting operation, you could argue to either make 2 classes MLTransformerBenchmarkable and MLEstimatorBenchmarkable (the latter even being a subclass of the former), or you could  argue to have one class general MLGeneralTransformerBenchmarkable, for which the the `fit` method is a no-op in case of transformers.

I do not have a strong preference for either pattern, so if you think that class should be restricted to Estimators, I will be happy to do that.